pegasus/sites.v3/code.org/views/homepage_below_hero_plane_banner.haml

```diff
@@ -1,22 +1,22 @@
 - if Homepage.show_professional_learning_banner(request)
   = inline_css 'homepage_below_hero_plane_banner.css'
 
+  - closing_soon_text = "2021 Professional Learning applications are closing soon! Sign up now."
+  - sign_up_text = "Sign up for Professional Learning! Middle and High School applications now open."
+  - text = Homepage.teacher_applications_closing_soon() ? closing_soon_text : sign_up_text
```

Comment:
Nit: might be nice to just have a method like `Homepage.get_teacher_application_text` instead of the `Homepage.teacher_applications_closing_soon()` that just returns the value of the string to display.